content/docs/cucumber/configuration.md

```diff
@@ -231,7 +231,48 @@ This is just a convention though; Cucumber will pick them up from any file{{% te
 # Profiles
 
 {{% block "java" %}}
-Profiles are not available in Java.
+Profiles are not available in Java.  However, it is possible to set configuration options using [Maven profiles](https://maven.apache.org/guides/introduction/introduction-to-profiles.html).
+
+For instance, we can configure separate profiles for scenarios which are to be run in separate environments like so:
+
+``` xml
+    <profiles>
+        <profile>
+          <id>dev</id>
+            <properties>
+                <cucumber.options>--tags @dev</cucumber.options>
```

Comment:
When I'm using multiple tags this may interact with mavens property interpolation. If I am not mistaken `--tags @dev and not @ignore` will be seen as the property `dev and not `. 

https://maven.apache.org/shared/maven-filtering/

Reply:
I've never tried this but it may be possible to avoid the interpolation by using the html entity `&#64;` for `@` instead.

Reply:
Let's make sure to use an example that will not lead to further questions.

Reply:
I was able to work around this by surrounding the arguments used for --tags in quotes.  Seemed to work fine when I tested it with Maven

Reply:
Great!